dlx_rest/static/js/validation.js

```diff
@@ -875,6 +875,26 @@ export const validationData = {
             "validSubfields": ["a", "g", "t"],
             "defaultSubfields": ["a"] 
         },
+        "506": {
+            "name": "Access restriction note",
+            "required": false,
+            "repeatable": false,
+            "validIndicators1": [],
+            "validIndicators2": [],
+            "requiredSubfields": [],
+            "validSubfields": ["a", "b", "f", "g"],
+            "defaultSubfields": ["a", "b", "f"],
+            "validStrings": {
+                "a": ["Restricted distribution","Confidential","Embargoed","Withdrawn","De-restricted"],
+                "f": ["Access limited to authorized users", "Access limited on-site", "Access limited on-site and to authorized users"]
+            },
+            "_validRegex": {
+                "g": ["^\d{4}-(0[1-9]|1[0-2])-(0[1-9]|[12]\d|3[01])$"] // should be a date yyyy-mm-dd check my regex.
+            },
+            "isDate": {
```

Comment:
This field is tells the validation checker to validate the value as a date using a JS datetime object. Therefore the date doesn't have to be validated by regex.

https://github.com/dag-hammarskjold-library/dlx-rest/blob/3f294b4d6127e29f5b09d9cc52aa6d71f1c04904/dlx_rest/static/js/jmarc.mjs#L111